setup.py

```diff
@@ -17,8 +17,7 @@
 if __name__ == "__main__":
     setup(name='coala-quickstart',
           version='0.4.0',
-          description='A quickstart tool for for coala '
-                      '(Code Analysis Application)',
+          description='A quickstart tool for for coala ',
```

Comment:
Can you also edit "tool ```for for``` coala" :smile: :+1:

Reply:
oh sorry I didn't see that